mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/result/MapperResultProducer.java

```diff
@@ -103,5 +103,5 @@
    * declares them, or wrapped into a {@link RuntimeException} otherwise.
    */
   @Nullable
-  Object wrapError(@NonNull Throwable error) throws Exception;
+  Object wrapError(@NonNull Exception e) throws Exception;
```

Comment:
Could be made default.

Reply:
How? The implementation will be specific to each custom result, for example `ListenableFuture<EntityT>` vs `Flowable<EntityT>`.

Reply:
By throwing:

```
  @Nullable
  default Object wrapError(@NonNull Exception e) throws Exception {
    throw e;
  }
```

Reply:
Throwing is not appropriate for any future or reactive type.